mmtrack/models/trackers/quasi_dense_embed_tracker.py

```diff
@@ -200,7 +200,7 @@ def track(self, img_metas, feats, model, bboxes, labels, frame_id):
                 np.zeros((0, 6), dtype=np.float32)
                 for _ in range(model.detector.roi_head.bbox_head.num_classes)
             ]
-            return track_bboxes
+            return track_bboxes, None, None
```

Comment:
Follow [this](https://github.com/open-mmlab/mmtracking/blob/master/mmtrack/models/trackers/masktrack_rcnn_tracker.py#L129) to return empty track bboxes.